src/components/pager/js/Pager.js

```diff
@@ -295,8 +295,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     });
 
     fluid.pager.previousNext.update = function (that, disabledStyle, newModel) {
-        that.previous.toggleClass(disabledStyle, newModel.pageIndex === 0);
-        that.next.toggleClass(disabledStyle, newModel.pageIndex === newModel.pageCount - 1);
+        var previousToggle = (newModel.pageIndex === 0);
+        var nextToggle = (newModel.pageIndex === newModel.pageCount - 1);
+        that.previous.toggleClass(disabledStyle, previousToggle);
+        that.next.toggleClass(disabledStyle, nextToggle);
+        that.previous.children('a').attr('aria-disabled', previousToggle);
+        that.next.children('a').attr('aria-disabled', nextToggle);
```

Comment:
I tried doing what you told I removed the that.previous.children("a").attr(...) and put that.previous.attr(...) but as a result of this when I use the keyboard to put focus on the disabled link, screen reader says "internal link disabled", but when i click with mouse on the disabled link it focuses on the link inside and says "internal link" only. Though I have pushed accordingly to what you said.